aptos-move/aptos-validator-interface/src/storage_interface.rs

```diff
@@ -94,16 +94,16 @@ impl AptosValidatorInterface for DBDebuggerInterface {
         unimplemented!();
     }
 
-    async fn get_latest_version(&self) -> Result<Version> {
-        self.0.get_latest_version().map_err(Into::into)
+    async fn get_committed_version(&self) -> Result<Version> {
+        self.0.get_committed_version().map_err(Into::into)
     }
 
     async fn get_version_by_account_sequence(
         &self,
         account: AccountAddress,
         seq: u64,
     ) -> Result<Option<Version>> {
-        let ledger_version = self.get_latest_version().await?;
+        let ledger_version = self.get_committed_version().await?;
```

Comment:
we want synced_version here?

Reply:
the "validator-interface" didn't expose "synced" version. I think we don't need either..

Reply:
I'd argue they should all be "synced" version.

Reply:
well, you can't safely get block info from a synced version, for one thing. I'd rather expose it to state sync only.

Reply:
ok... @JoshLind when state sync on block boundary?